src/vscode-dts/vscode.proposed.quickPickItemTooltip.d.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	// https://github.com/microsoft/vscode/issues/73904
+
+	export interface QuickPickItem {
+		/**
+		 * An optional flag to sort the final results by index of first query match in label. Defaults to true.
+		 */
+		tooltip?: string | MarkdownString;
```

Comment:
@TylerLeonhardt The description and issue number seems incorrect?

Reply:
Fixed!